# Generated by Django 2.2.5 on 2019-09-17 11:54

from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        ('authentication', '0016_ownershiprecord'),
    ]

    operations = [
        migrations.CreateModel(
            name='AllianceContact',
            fields=[
                ('contact_id', models.IntegerField(primary_key=True, serialize=False)),
                ('contact_type', models.CharField(max_length=32)),
                ('standing', models.FloatField()),
            ],
        ),
        migrations.CreateModel(
            name='AllianceManager',
            fields=[
                ('character', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, primary_key=True, serialize=False, to='authentication.CharacterOwnership')),
                ('version_hash', models.CharField(default=None, max_length=32, null=True)),
                ('last_sync', models.DateTimeField(default=None, null=True)),
            ],
        ),
        migrations.CreateModel(
            name='SyncedCharacter',
            fields=[
                ('character', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, primary_key=True, serialize=False, to='authentication.CharacterOwnership')),
                ('version_hash', models.CharField(default=None, max_length=32, null=True)),
                ('last_error', models.IntegerField(choices=[(0, 'No error'), (1, 'Invalid token'), (99, 'Unknown error')], default=0)),
                ('last_sync', models.DateTimeField(default=None, null=True)),
            ],
        ),
    ]
